tools/clang/test/HLSLFileCheck/hlsl/functions/arguments/outputArray-strictudt.hlsl

```diff
@@ -0,0 +1,26 @@
+// RUN: %dxc -E main -T vs_6_0 -strict-udt-casting %s | FileCheck %s
+
+// CHECK: switch
```

Comment:
Not sure what this is meant to check

Reply:
It's a modified version of an existing test for something else, that runs with this flag on because it relied on implicit UDT casting.  I thought leaving the original version is helpful to make sure the prior behavior continues to work, and the new test makes sure we didn't break something when forced to explicitly cast in the same scenario.